src/main/scala/org/ergoplatform/wallet/interpreter/ErgoProvingInterpreter.scala

```diff
@@ -60,6 +60,8 @@ class ErgoProvingInterpreter(val secretKeys: IndexedSeq[ExtendedSecretKey], para
             ContextExtension.empty
           )
 
+          val maxCost = params.maxBlockCost
```

Comment:
@greenhat the cost should passed to `context` (since it no longer passed to Interpreter)

Reply:
Thanks! Fixed.